doc/metrics.md

```diff
@@ -151,9 +155,12 @@ scrape_configs:
       #      is not covered by the certificate (not in the SAN list)
       server_name: 'foo'
 ```
+```{note}
```

Comment:
```suggestion
````{note}
```

Usually 3 backticks are enough, but not in this case since the note include another directive.
So this one is trickier than usual. :wink: